plugins/modules/monitoring/monit.py

```diff
@@ -13,24 +13,29 @@
 module: monit
 short_description: Manage the state of a program monitored via Monit
 description:
-     - Manage the state of a program monitored via I(Monit)
+    - Manage the state of a program monitored via I(Monit)
```

Comment:
```suggestion
    - Manage the state of a program monitored via I(Monit).
```
`short_description` shouldn't have a final dot but `description` must have